### ISSUE TITLE
update css on images

### ISSUE THREAD

**Issue by Manya200** [JayPokharkar/Hotel-Managemet-System#2]
Fluctuations in images , rendering throughout the process , can be more responsive.

**Comment by janhaviiiiiii7** [comment]
i want to work on this project please kindly assign me for further changes.

**Comment by jayboy-jp** [comment]
> i want to work on this project please kindly assign me for further changes.

Yeah go on.. you're free to do code and have fun.. yeah I'll assign.. good luck!..
